internal/runbits/requirements/requirements.go

```diff
@@ -297,10 +297,6 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 	//	return errs.Wrap(err, "Could not get remote build expr")
 	//}
 
-	if err := commitmediator.Set(r.Project, commitID.String()); err != nil {
-		return locale.WrapError(err, "err_package_update_commit_id")
-	}
-
 	// Note: a commit ID file needs to exist at this point.
```

Comment:
I think we need to move this commented out chunk of code to below the new location of `commitmediator.Set()`. As the top comment says, a commitID file needs to exist at this point.